pkg/clients/ibmcloud.go

```diff
@@ -46,6 +46,8 @@ import (
 	rcv2 "github.com/IBM/platform-services-go-sdk/resourcecontrollerv2"
 	rmgrv2 "github.com/IBM/platform-services-go-sdk/resourcemanagerv2"
```

Comment:
just to be consistent with formatting for imports, I think it's best to remove the empty line between line 47 and 49

Reply:
The code looks good for adding the support for the new API here in ibmcloud.go